pyproject.toml

```diff
@@ -49,8 +49,7 @@ pinecone-client = "^3.0.3"
 pymongo = "^4.6.0"
 supabase = "^2.3.0"
 certifi = ">=2023.11.17,<2025.0.0"
-psycopg = "^3.1.9"
-psycopg-binary = "^3.1.9"
```

Comment:
Removing this causes an error, I think. That's why it was two separate declarations

Reply:
I'm not sure. I remember we had both for a reason, I just can't remember what it was